### DISCUSSION TITLE
Consumer not connecting to available broker after broker restart

### DISCUSSION THREAD

**Discussion by amitsingh-zscaler** [confluentinc/confluent-kafka-go#828] [Q&A] [answered]
Configuration:
I  have 10 brokers with 3 partition on each topic. Replication factor on topic is also set to 3.

Scenario:
I have consumer which is working properly, suddenly a broker(broker-1) goes down. The consumer stops getting events and starts throwing below exception.
`ERROR occurred while consuming events, Error: error: Local: Broker transport failure: *******-*****.elb.us-**-2.amazonaws.com:9094/4: Connect to ipv4#**.**.**.**:9094 failed: Operation timed out (after 129624ms in state CONNECT)"}`

It starts working only when the broker(broker-1) is up.

Expectation:
Consumer should connect to other available brokers instead of waiting for old broker(broker-1) to come up since the replication is set to 3.

**Comment by elibarel10** [comment, marked as answer]
release 1.9.2 should fix this.
("The AdminClient poll loop no longer terminates when a request results in an error")
